src/isDeepEqual.ts

```diff
@@ -1,11 +1,11 @@
 import { purry } from "./purry";
 
 /**
- * Performs a deep *semantic* comparison between two values to determine if they
- * are equivalent. For primitive values this is equivalent to `===`, for arrays
- * the check would be performed on every item recursively, in order, and for
- * objects all props will be compared recursively. The built-in Date and RegExp
- * are special-cased and will be compared by their values.
+ * Performs a _deep_ *semantic* comparison between two values to determine if
+ * they are equivalent. For primitive values this is equivalent to `===`, for
+ * arrays the check would be performed on every item recursively, in order, and
+ * for objects all props will be compared recursively. The built-in Date and
+ * RegExp are special-cased and will be compared by their values.
```

Comment:
**question:** should we mention `Set` and `Map` here?

i now realize that we added that in some PR but didn't update this description

Reply:
I think it's assumed that they work. We had them in the important block below as stuff that doesn't work because it would be implied that they do if we didn't.